rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,501 @@
+imaging prototype. Additional information on cellular geometries and QC
+will be provided as a supplementary metadata zarr archive.
+
+However, other imaging experiments will likely require extending how
+features (genes) are named to support a broader set of features
+(eg. proteins and cellular components). Currently, the DCP standardizes
+around GENCODE gene IDs and HGNC gene symbols. Image based proteomics
+will require protein identifiers such as UNIPROT, and other use cases
+that measure sub-cellular components will likely require more general
+cellular component information (e.g. "endoplasmic reticulum").
+
+### **Acceptance Criteria**
+
+Except where listed ("responsible team"), the starfish and OME teams
+will do the majority of project management and execution for the
+following deliverables. They will coordinate and collaborate with DCP
+partners ("contact person") to validate design correctness and minimize
+code changes and their impact.
+
+| Component/Deliverable                                 	| Responsible Team/Service 	| No earlier than 	| Contact Person	|
+|-------------------------------------------------------	|--------------------------	|-----------------	|---------------	|
+| HCA Metadata Draft                                    	| EBI                      	| Complete        	| Z. Perova     	|
+| Test Datasets Available                               	| starfish                 	| Complete        	| A. Carr       	|
+| Submission Datasets Available                         	| starfish                 	| Q2 2019         	| A. Carr       	|
+| Analysis Pipelines                                    	| Broad                    	| Q2 2019         	| ?             	|
+| Analysis & Adapter Pipelines                          	| Broad                    	| Q2 2019         	| A. Carr       	|
+| Lira subscription                                     	| Broad                    	| TBD             	| S. Ehsan      	|
+| Data Analysis Service starfish Pipeline Tests         	| Broad                    	| Q2 2019         	| G. Grant      	|
+| Specification for input format                        	| starfish                 	| Complete        	| J. Moore      	|
+| Specification for output format                       	| starfish                 	| Q1 2019         	| A. Carr       	|
+| Input validation tool                                 	| starfish, OME            	| Q3 2019         	| J. Moore      	|
+| Output validation tool                                	| starfish, OME            	| Q2 2019         	| A. Carr       	|
+| Ingest support for the nested arrays of channels      	| EBI                      	| ?               	| Z. Perova             	|
+| [Ability to describe files that belong together][a]   	| ?                        	| ?               	| ?             	|
+| [Upload and download of subdirectories][b]             	| ?                        	| ?               	| ?             	|
```

Comment:
```suggestion
| [Upload and download of subdirectories][b]             	| ?                        	| ?               	| H. Schmidt             	|

Reply:
@hannes-ucsc if you're not the right contact for this, could you suggest someone else?